_im-plugin/notifications-settings.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: default
+title: Configuring notifications settings
+parent: Managing indexes
+nav_order: 100
+---
+
+## Configuring notifications settings
+
+Introduced 2.8
+{: .label .label-purple }
+
+Notifications settings allow users to control how they receive notifications about index events. You can use either OpenSearch Dashboards or the REST API to configure notifications. See [Notifications]({{site.url}}{{site.baseurl}}/observing-your-data/notifications/index/) and [Configuring notifications through Dashboards]({{site.url}}{{site.baseurl}}/dashboards/im-dashboards/notifications/) for further details. 
+
+Configuring notifications settings is useful for long-running index operations. When creating long-running operation notifications using the API, you can configure the `lron_config` setting in two ways:
+
+- As a one-time task: If you pass a `task_id` in the `lron_config` object, the task is one-time and the setting automatically deletes when the task ends.
+- As a global and persistent task: If you pass an `action_name` in the `lron_config` object, the task is global, persistent, and applies to all operations of this action type.
```

Comment:
To be precise, If you pass an `action_name` and not pass the "task_id", then it's global and persistent